emsdk/patches/0007-Fix-lookupPath-when-applied-to-a-symlink-loop.patch

```diff
@@ -0,0 +1,31 @@
+From c1120e713b28110945baeffdbd924aeac2ea4f26 Mon Sep 17 00:00:00 2001
+From: Hood Chatham <roberthoodchatham@gmail.com>
+Date: Wed, 2 Mar 2022 13:44:14 -0800
+Subject: [PATCH 7/7] Fix lookupPath when applied to a symlink loop
+
+The following code leads to an infinite loop in lookupPath:
+
+FS.symlink("linkX/inside","/linkX");
+FS.lookupPath("/linkX", {follow:true});
+
+This patch fixes it.
+---
+ src/library_fs.js | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/library_fs.js b/src/library_fs.js
+index 519fab972..ef91655f2 100644
+--- a/src/library_fs.js
++++ b/src/library_fs.js
+@@ -169,7 +169,7 @@ FS.staticInit();` +
+             var link = FS.readlink(current_path);
+             current_path = PATH_FS.resolve(PATH.dirname(current_path), link);
+ 
+-            var lookup = FS.lookupPath(current_path, { recurse_count: opts.recurse_count });
++            var lookup = FS.lookupPath(current_path, { recurse_count: opts.recurse_count + 1 });
```

Comment:
Is there an upstream issue?

Reply:
https://github.com/emscripten-core/emscripten/pull/16410